helix-term/src/ui/prompt.rs

```diff
@@ -234,8 +234,8 @@ impl Prompt {
             }
             Movement::StartOfLine => 0,
             Movement::EndOfLine => {
```

Comment:
I think this branch can just return `self.line.len()` instead

Reply:
Thank you. That worked and I've fixed it.